### PR TITLE
added forgot & update password pages

### PR DESCRIPTION
**Pull request by Kelvin229**
- Added update password and forgot password pages
- Corrected SignIn -> Login, for clarity

Lmk if you have any questions!

### CODE REVIEW

File: frontend/src/containers/Auth/SignUp.tsx [hunk 1]
```diff
@@ -162,7 +162,7 @@ const SignUp = () => {
         </div>
         
         <div className={styles.authTermPolicy}>
-          By signing up, you agree to Quiz Qrafter's{" "}
+          By Loging up, you agree to Quiz Qrafter's{" "}
```
Comment (gideonoludeyi) [inline frontend/src/containers/Auth/SignUp.tsx]:
```suggestion
          By logging in, you agree to Quiz Qrafter's{" "}
```

File: frontend/src/services/auth/types.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 export interface AuthService {
   currentUser(): Promise<User | null>;
   signUp(info: SignUpInfo): Promise<void>;
-  signIn(credentials: LoginCredentials): Promise<User>;
+  Login(credentials: LoginCredentials): Promise<User>;
```
Comment (gideonoludeyi) [inline frontend/src/services/auth/types.ts]: Can we make this to camelCase (ie. `login`) to keep it consistent with other method names?

File: frontend/src/services/auth/context.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ type AuthContextValue = {
   error: Error | null;
   authenticated: boolean;
   signUp(email: string, password: string, fullname: string): void;
-  signIn(email: string, password: string): void;
+  Login(email: string, password: string): void;
```
Comment (gideonoludeyi) [inline frontend/src/services/auth/context.tsx]: camelCase here too

File: frontend/src/services/auth/types.ts (hunk 2 above)
Comment (gideonoludeyi) [reply frontend/src/services/auth/types.ts]: You might also have to update where the `Login` method is being called to `login`

File: frontend/src/containers/Auth/Login.tsx [hunk 4]
```diff
@@ -4,8 +4,8 @@ import Logo from "../../assets/quiz_qrafter_logo_dark.svg"; // Use a regular ima
 import { useAuth } from "../../services/auth";
 import styles from "./auth.module.css";
 
-const SignIn = () => {
-  const { authenticated, signIn } = useAuth();
+const Login = () => {
+  const { authenticated, Login } = useAuth();
```
Comment (gideonoludeyi) [inline frontend/src/containers/Auth/Login.tsx]: Seems you missed here and [line 61](https://github.com/QuizQrafter/quiz-qrafter/pull/13/files#diff-b0259f43843454dea32956dc5727215037ef77b8ab9a768a8e3d442916360794R61) when renaming `Login` to `login`.

I was getting this error as a result
```ts
TS2339: Property 'Login' does not exist on type 'AuthContextValue'.
     6 |
     7 | const Login = () => {
  >  8 |   const { authenticated, Login } = useAuth();
       |                          ^^^^^
     9 |   const [email, setEmail] = useState("");
    10 |   const [password, setPassword] = useState("");
    11 |   const [error,setError] = useState<Record<string, string>>({});
```

Comment (Kelvin229) [reply frontend/src/containers/Auth/Login.tsx]: My bad, I've made a new commit to resolve this issue.

**Review by gideonoludeyi — APPROVED** [review]

**Pull request merged**